data/cameras.xml

```diff
@@ -491,7 +521,7 @@
 			<Hint name="invert_sraw_wb" value=""/>
 		</Hints>
 	</Camera>
-	<Camera make="Canon" model="Canon EOS 5DS R" decoder_version="6">
+	<Camera make="Canon" model="Canon EOS 5DS R" decoder_version="6" supported="no">
```

Comment:
Please note, that not only darktable uses RawSpeed, but other software uses it too.
Any LibRaw-based software (dozens directly and many indirectly, via FreeImage or other wrappers) may use RawSpeed as decoder.

Disabling right decoded camera because darktable cannot use it - IS A VERY BAD IDEA.